hummingbot/connector/utils.py

```diff
@@ -90,6 +90,15 @@ def get_new_client_order_id(
     return client_order_id
 
 
+def get_new_numeric_client_order_id(nonce_creator: NonceCreator, max_id_bit_count: Optional[int] = None) -> int:
+    local_ip_host_part = "".join(socket.gethostbyname(socket.gethostname()).split(".")[-2:])
+    client_order_id = int(f"{local_ip_host_part}{os.getpid()}{nonce_creator.get_tracking_nonce()}")
+    if max_id_bit_count:
+        max_int = 2 ** max_id_bit_count - 1
+        client_order_id &= max_int
+    return client_order_id
```

Comment:
Ok, I will change to this approach to the host part.

I went with the approach of limiting the bits in the instance ID because Eve is our first numeric-only candidate, and it limits the ID to a 64 bit integer (e.g. presumably `18446744073709551615` will work, but `28446744073709551615` wont, even though they both have the same char len). So, bit-wise limitation allows us to more easily capture the full range of IDs with the same effect